src/main/java/com/de_alone/dokkang/controllers/LectureController.java

```diff
@@ -0,0 +1,31 @@
+package com.de_alone.dokkang.controllers;
+
+import com.de_alone.dokkang.models.Lecture;
+import com.de_alone.dokkang.payload.response.LectureResponse;
+import com.de_alone.dokkang.repository.LectureRepository;
+import com.de_alone.dokkang.security.jwt.JwtUtils;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@CrossOrigin(origins = "*", maxAge = 3600)
+@RestController
+@RequestMapping("/lecture")
```

Comment:
```suggestion
@RequestMapping("/lectures")
```
모든 강의 목록 가져오는 endpoint는 lectures (lecture + GET method는 특정 lecture 하나를 가져올 때)